plugin.rb

```diff
@@ -122,6 +119,12 @@ def self.allowed_group_ids
   load File.expand_path('../app/jobs/scheduled/update_user_counts_for_chat_channels.rb', __FILE__)
   load File.expand_path('../app/services/chat_publisher.rb', __FILE__)
 
+  if ENV['ALLOW_DEV_POPULATE'] == '1'
```

Comment:
```suggestion
if Discourse.allow_dev_populate?
```

https://github.com/discourse/discourse/blob/14f61c57847b6d0a861653e2f479947ad9b68962/lib/discourse.rb#L1026-L1028